packages/synapse-react-client/src/components/SynapseTable/usePrefetchTableData.ts

```diff
@@ -81,79 +87,76 @@ function usePrefetchFileHandleData() {
   )
 }
 
-function useGetEntitiesInTable() {
-  const entity = useAtomValue(tableQueryEntityAtom)
-  const data = useAtomValue(tableQueryDataAtom)
+function useGetEntitiesInTable(rowSet: RowSet) {
```

Comment:
Updated to pass RowSet as an argument, and use queryContext + react-query instead of Jotai. No logical changes